Integrations/RiskSense/RiskSense.py

```diff
@@ -0,0 +1,1975 @@
+        return 'No unique cves found for the given argument.', {}, {}
+
+    hr = ''
+    ec = {}  # type: Dict[str, Any]
+    if resp and '_embedded' in resp.keys():
+        host_findings = resp.get('_embedded', {}).get('hostFindings', [])
+        host_findings_context = []  # type: List[Dict[str, Any]]
+
+        for host_finding in host_findings:
+            vulnerabilities = host_finding.get('vulnerabilities', {}).get('vulnInfoList', [])
+
+            if len(vulnerabilities) == 0:
+                return 'No vulnerabilities found for the given argument.', {}, {}
+
+            # Human readable.
+            hr += '### Client: ' + client_name + '\n'
+            hr += tableToMarkdown('Vulnerabilities found:', get_vulnerabilities_hr(vulnerabilities),
+                                  ['Name', 'V2/Score', 'Attack Vector', 'Attack Complexity', 'Authentication',
+                                   'Confidentiality Impact', 'Integrity Impact', 'Availability Impact', 'Summary'])
+
+            #  Prepare context data
+            host_findings_context.extend(get_unique_cves_context(vulnerabilities))
+
+        ec = {
+            'RiskSense.HostFinding.Vulnerability(val.Cve == obj.Cve)': host_findings_context
+        }
+    else:
+        hr += 'No Vulnerabilities found for a given argument.'
+
+    return hr, ec, resp
+
+
+def get_open_host_findings_command(client, args):
+    """
+    Retrieves information about open host findings based on CHMLI and store it in context data.
+
+    :param client: object of client class
+    :param args: Demisto arguments provided by user
+    :return: standard output
+    """
+    client_detail = get_client_detail_from_context(client)
+    client_id = client_detail['Id']
+    client_name = client_detail['ClientName']
+    url_suffix = '/' + str(client_id) + '/hostFinding/search'
+
+    data = prepare_filter_payload(args, 'detail')
+
+    # adding additional filters
+    data = add_filter_to_request(data, 'generic_state', 'false', 'EXACT', 'open')
+
+    # making final API call
+    resp = client.http_request('POST', url_suffix=url_suffix, json_data=data)
+
+    page_number, total_element, total_pages = fetch_page_details(resp)
+
+    if total_element == 0:
+        return 'No open host finding(s) details found for given argument(s).', {}, {}
+    if page_number >= total_pages:
+        raise ValueError('Invalid page navigation.')
+
+    ec = {}  # type: Dict[str, Any]
+    hr = ''
+
+    if resp and '_embedded' in resp.keys():
+        resp_list_hostfinding = resp.get('_embedded', {}).get('hostFindings', [])
+
+        host_finding_details_hr = []  # type: List[Dict[str, Any]]
+        risksense_host_finding_context = []  # type: List[Dict[str, Any]]
+
+        for resp_hostfinding in resp_list_hostfinding:
+            host_finding_details_hr.append(get_host_finding_hr(resp_hostfinding))
+            risksense_host_finding_context.append(get_risksense_host_finding_context(resp_hostfinding))
+
+      
```

Comment:
this command uses the exact same logic as the opened one, it will be easier to maintain just one function - not a must.

Reply:
@ronykoz this has been taken care